app/containers/HeaderButton/HeaderButtonContainer.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { StyleSheet, View } from 'react-native';
 
 interface IHeaderButtonContainer {
-	children: React.ReactNode;
+	children: React.ReactElement | React.ReactElement[] | null;
```

Comment:
For some reason this is raising an error on other screens (That's why probably lint is failing)